tests/ci_visibility/test_ci_visibility.py

```diff
@@ -492,6 +496,7 @@ def test_civisibilitywriter_coverage_agentless_url():
 
 
 def test_civisibilitywriter_coverage_agentless_with_intake_url_param():
+    ddtrace.internal.ci_visibility.writer.config._ci_visibility_agentless_url = ""
```

Comment:
nit: We could use tests.utils.override_global_config(...) here. Wrapping this test in a contextmanager could be cleaner

Reply:
For some reason that approach didn't work in this test